src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -769,6 +763,88 @@ Value emitScalarOpFor<ONNXSignOp>(ConversionPatternRewriter &rewriter,
   return create.math.select(zeroPredicate, zero, plusSelect);
 }
 
+//===----------------------------------------------------------------------===//
+// Scalar unary ops for lowering ONNXErfOp
+//===----------------------------------------------------------------------===//
+template <>
+struct ScalarOp<ONNXErfOp> {
+  using FOp = CustomScalarOp;
+  using IOp = NotSuportedScalarOp;
+};
+
+template <>
+double analyzeSimdFor<ONNXErfOp>(Type t, int64_t &von, int64_t &son) {
+  return simdAnalysis(
+      {GenericOps::ArithmeticGop, GenericOps::CompareGop, GenericOps::DivGop,
+          GenericOps::MulGop, GenericOps::SelectGop, GenericOps::AbsGop,
+          GenericOps::ExpGop},
+      {6, 2, 1, 11, 2, 1, 1}, t, von, son);
+}
```

Comment:
I would use FMAs in the code and here:

arithmetic 2
compare 1
div 1
mul 4
FMA 5
select 1
abs 1
exp 1

Reply:
compare 1 arith.cmpf
     dievp 1 arith.divf
      6 vector.fma
      4 arith.mulf
      1 arith.select
      1 math.absf
      1 math.exp

Reply:
Two mul are change to arithmetic ops (by Fix  with (-1.0 * x) to (0.0 -x)).

arithmetic 2 arith.subf
compare 1 arith.cmpf
div 1 arith.divf
fma 6 vector.fma
mul 2 arith.mulf
select 1 arith.select
abs 1 math.absf
exp 1 math.exp